src/messages/message_raw_crypto.rs

```diff
@@ -218,9 +214,9 @@ impl Message {
     /// Verifies signature and returns payload message on verification success.
     /// `Err` return if signature invalid or data is malformed.
     /// Expects Jws's payload to be a valid serialized `Message` and base64_url encoded.
-    pub fn verify_value(jws: &Value, key: &[u8]) -> Result<Message, Error> {
+    pub fn verify_value(jws: &Value, signing_sender_public_key: &[u8]) -> Result<Message, Error> {
```

Comment:
```suggestion
    ///
    /// # Arguments
    ///
    /// * `jws` - to be verified jws message as json `Value` object
    ///
    /// * `signing_sender_public_key` - optional public key used for verification, if `None` it will try to resolve the did in the `kid` field
    pub fn verify_value(jws: &Value, signing_sender_public_key: &[u8]) -> Result<Message, Error> {
```